### ISSUE TITLE
Papel changes copy pasted characters when it becomes paper bundle.

### ISSUE THREAD

**Issue by Pisidora** [ParadiseSS13/Paradise#21790]
**Issue Description**:
When copypasting from Google Docs (Arial 11) to the game certain characters ( the ' and " ) become half corrupted and change to random things when the papel is added to a papel bundle, not when copied in a photocopier though. 

**What did you expect to happen**:
I expected to either bug exactly when I write it down or not bug at all. I had a feeling that copypasting with the font and number may bug though.

**What happened instead**:
When added the papel to a bundle the characters ' and " become almost random characters, like the euro symbol.

**Why is this bad/What are the consequences**:
It's not top priority but it is annoying at sharing own work, it may acidentaly help not copying outside work intro the game though, but it just adds more work to do (rewrite every ' and " ). It's just annoying. 

**Steps to reproduce the problem**:
Go to google docs, I haven't tried with any other word processor, but probably any works);
Write something using either the characters ' or ", I haven't found any other that gives problems;
Copy and paste it into a papel via pen
Get any other piece of papel and add it to the first one.
Read the papel.

**When did the problem start happening**:
Today 23 of July in 2023. I haven't tried to copy my work intro the game before so I don't really know when it started.

**Extra information**:
I have only tried with google docs, arial 10. I strongly believe the problem comes from copying and pasting the font itself, since if I rewrite every malfuncional character individually then it works fine.
I tried to see if the problem is in actualizing the papel but no, I have even copied it via photocopier and the problem only appears once it becomes a paper bundle.
Also, it works fine again if the papel is separated from the bundle.

![bug report 0](https://github.com/ParadiseSS13/Paradise/assets/113777596/3037d3f8-86be-4fc6-b559-79ee4df7df6e)
![bug report 1](https://github.com/ParadiseSS13/Paradise/assets/113777596/03d5e96e-f516-46f0-b769-98d0f417cf69)

**Comment by Contrabang** [comment]
Yeah, this is because it doesnt properly format, ideally those quotes should be switched into the actual quote symbols. Google doc does some weird stuff with turning normal quotes into forward and backward quotes

**Comment by Pisidora** [comment]
Okey :>. It still toke me a while to write this but if it's useles then I'll move on.

**Comment by Pisidora** [comment]
But question, if it's google docs fault, the why does it only happen when it is a paper bundle and not with plain papel?

**Comment by Contrabang** [comment]
Paper bundle has better handling for stuff like this. Its still an issue we can fix, but the fault is because of weird character from google docs. It'll probably get fixed someday. Thank you for the report anyways, very helpful!
